### PR TITLE
#FT-01 Design main pages

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This Pull Request aims to add design for the landing page of the Medical core

# Description of what this task covered

- [x] design home page
- [x] design product page
- [x] designs pricing page
- [x] design contact page
- [x] design blog page
- [x] designs blog post page

# How this may be tested?

1. Clone the repo
4. Navigate into special-tb3
3. Check out to the appropriate branch(ft/main-page)
4. Open index.html in your browser or you can use your live server extension

# Screenshots (If appropriate/applicable)

### CODE REVIEW

File: page/blog-post.html [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../dist/output.css">
+    <title>Blog post</title>
+</head>
+<body>
+    <div class="header">
+        <div class="header__nav">
+            <div class="logo">
```
Comment (edson-pro) [inline page/blog-post.html]: must differentiante footer logo with nav logo

**Review by edson-pro — CHANGES_REQUESTED** [review]
DLGTM

File: page/blog-post.html (hunk 1 above)
Comment (chrisostome01) [reply page/blog-post.html]: let me work on that 👍

**Review by edson-pro — APPROVED** [review]

File: style/main.css [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+.input-group--size-small {
+    @apply w-[300px]
+}
+
+.button--bg-default {
+    @apply bg-transparent
+}
+
+.input {
+    outline: none;
```
Comment (IbrahimBagalwa) [inline style/main.css]: outline none exists in tailwnd ... no need to use it as css

File: style/main.css [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+.scale-card {
+    @apply flex flex-col gap-7
+}
+
+.scale-card__item {
+    @apply flex gap-3 flex-col
+}
+
+.scale-card__title {
+    @apply text-primary-500 text-[24px] leading-[33.6px] font-bold
```
Comment (IbrahimBagalwa) [inline style/main.css]: this is a custom color no need of this opacity

File: style/main.css [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+}
+/* ========== End: scale-card ==================== */
+
+/* ========== start: gridient-card =============== */
+.gridient-card {
+    @apply w-full min-h-full relative mt-[2rem];
+}
+
+.gridient-card__gradient {
+    background: radial-gradient(483.9% 2719.65% at -49.5% -250%, #D798E1 17.55%, #9BFFA5 27.56%, #AED3FF 49.89%, #C9D4EF 56.53%, #CACFFA 65.69%);
```
Comment (IbrahimBagalwa) [inline style/main.css]: this is not tailwindcss bro

File: style/main.css [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+.header__nav {
+    @apply flex justify-between 
+}
+
+.header__content {
+    @apply relative flex flex-col items-center justify-center gap-12 px-28;
+}
+
+.header__gradient {
+    background: radial-gradient(483.9% 2719.65% at -49.5% -250%, #D798E1 17.55%, #9BFFA5 27.56%, #AED3FF 49.89%, #C9D4EF 56.53%, #CACFFA 65.69%);
```
Comment (IbrahimBagalwa) [inline style/main.css]: ## css and tailwind combine
you are trying to combine css and tailwind, `background` make this like a custom css

File: style/main.css [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+    @apply px-6 py-3 font-bold
+}
+
+.button--bg-dark {
+    @apply bg-primary-500 text-white rounded-[12px]
+}
+
+.button--outline-dark {
+    @apply rounded-[12px];
+    border: 1px solid rgb(194, 191, 191)
```
Comment (IbrahimBagalwa) [inline style/main.css]: even here `border`

File: style/main.css [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+    @apply font-bold w-full 
+}
+
+.section-header__text {
+    @apply font-normal text-[18px] leading-[27px] w-full;
+}
+
+.section-header__higlight {
+    @apply font-bold text-[24px] w-full;
+    background: linear-gradient(53.94deg, #373FFF 15.02%, #3ACAF8 82.83%);
```
Comment (IbrahimBagalwa) [inline style/main.css]: even here `background`

File: style/main.css [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+}
+
+.user-card {
+    @apply grid grid-cols-3 gap-4 my-[8rem]
+}
+
+.card {
+    @apply flex items-center justify-center p-5 text-center flex-col gap-4 rounded;
+    border: 2px solid rgba(97, 153, 237, 0.26);
+    border-radius: 24px;
```
Comment (IbrahimBagalwa) [inline style/main.css]: you are using `rounded` and `border-radius` but i think here you are repeating your self. rounded in tailwind it means border-radius in css

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
## Great Job @sezeranoJchrisostome 

Great Job keep it up :)
implement the change and re-request the review again. 
### thank you

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job @sezeranoJchrisostome 
Keep it up man 👍
LGTM

File: index.html [hunk 9]
```diff
@@ -0,0 +1,453 @@
...
+                <li class="header__navigation-item" >
+                    <button class="header__button">
+                        View Pricing
+                    </button>
+                </li>
+            </ul>
+        </div>
+        <div class="header__content">
+            <div class="header__gradient"></div>
+            <div class="section-header">
```
Comment (IbrahimBagalwa) [inline index.html]: ## wrong name
referring to what the Tech lead told us you cannot have a nested block, is not a good practice here `section-header` in side the `header` try to fix it.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="section-header__title section-header__title--size-large">
+                    The Best Software to Grow <br> your Sales and Services
+                </div>
+                <div class="section-header__text">
```
Comment (IbrahimBagalwa) [inline index.html]: ### same issue 
`section-header__title` `section-header__title--size-large` `section-header__text`
here as well

File: index.html [hunk 11]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/arrow-right.svg" alt="arrow right">
+                    </div>
+                    <div class="scale__card-text">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
+                    </div>
+                </div>
+            </div>
+            <div class="scale-card__item">
```
Comment (IbrahimBagalwa) [inline index.html]: ## wrong name 
`scale-card__item` you are using the element outside its block.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,453 @@
...
```
Comment (IbrahimBagalwa) [inline index.html]: ### same issue
`scale-card__item` here as well

File: index.html [hunk 13]
```diff
@@ -0,0 +1,453 @@
...
+            </div>
+        </div>
+    </div>
+    
+    <div class="brands">
+        <img class="brand__logo" src="./assets/svgs/brands/brand1.svg" alt="brand icon">
+        <img class="brand__logo" src="./assets/svgs/brands/brand2.svg" alt="brand icon">
+        <img class="brand__logo" src="./assets/svgs/brands/brand3.svg" alt="brand icon">
+        <img class="brand__logo" src="./assets/svgs/brands/brand4.svg" alt="brand icon">
+        <img class="brand__logo" src="./assets/svgs/brands/brand5.svg" alt="brand icon">
```
Comment (IbrahimBagalwa) [inline index.html]: ## wrong name
`brand__logo` you are using element outside its block

File: index.html [hunk 14]
```diff
@@ -0,0 +1,453 @@
...
+    </div>
+    
+    <div class="scale">
+        <div class="scale-card">
```
Comment (IbrahimBagalwa) [inline index.html]: ## prohibition of nested block
`scale-card` nested block is not a good practice

File: index.html [hunk 15]
```diff
@@ -0,0 +1,453 @@
...
+            <div class="section-header section-header--posistion-left">
```
Comment (IbrahimBagalwa) [inline index.html]: ### same issue
`section-header` `section-header--posistion-left` 
here as well

File: index.html [hunk 16]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="section-header__higlight">
+                    Discover More
+                </div>
+                <div class="section-header__title section-header__title--size-small">
+                    Analyze your sales and marketing leads
+                </div>
```
Comment (IbrahimBagalwa) [inline index.html:189]: ### same issue
`section-header__higlight` `section-header__title` `section-header__title--size-small`
here as well

File: index.html [hunk 17]
```diff
@@ -0,0 +1,453 @@
...
+            </div>
+            <div class="scale__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.
+            </div>
+        </div>
+        <div class="scale-card">
```
Comment (IbrahimBagalwa) [inline index.html]: ## prohibation nested block
`scale-card` in `scale` make an element of `scale`

File: index.html [hunk 18]
```diff
@@ -0,0 +1,453 @@
...
+            <div class="scale-card__item">
```
Comment (IbrahimBagalwa) [inline index.html]: ### same issue
`scale-card__item` here as well

File: index.html [hunk 19]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/charticon.svg" alt="arrow right">
+                    </div>
+                    <div class="scale-card__content">
+                        <div class="scale-card__title">
+                            Sales Tracking
```
Comment (IbrahimBagalwa) [inline index.html:203]: `scale-card__content` `scale-card__title` here as well

File: index.html [hunk 20]
```diff
@@ -0,0 +1,453 @@
...
+                        </div>
+                        <div class="scale__card-text">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="scale-card__item">
```
Comment (IbrahimBagalwa) [inline index.html]: `scale-card__item` here as well

File: index.html [hunk 21]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/usericon.svg" alt="arrow right">
+                    </div>
+                    <div class="scale-card__content">
+                        <div class="scale-card__title">
+                            Project Management
```
Comment (IbrahimBagalwa) [inline index.html:218]: `scale-card__content` `scale-card__title` here as well

File: index.html [hunk 22]
```diff
@@ -0,0 +1,453 @@
...
+                        </div>
+                        <div class="scale__card-text">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="scale-card__item">
```
Comment (IbrahimBagalwa) [inline index.html]: `scale-card__item` here as well

File: index.html [hunk 23]
```diff
@@ -0,0 +1,453 @@
...
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/userchat.svg" alt="arrow right">
+                    </div>
+                    <div class="scale-card__content">
```
Comment (IbrahimBagalwa) [inline index.html]: the same issue `scale-card__title` here

File: index.html [hunk 24]
```diff
@@ -0,0 +1,453 @@
...
+                The stunning results our <br>
+                customers have experienced
+            </div>
+        </div>
+        <div class="testimony__card">
+            <div class="testimony__user-card">
+                <div class="testimony__coments">
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+                </div>
+                <img src="./assets/image/ceo.png" class="card__image" alt="user profile">
```
Comment (IbrahimBagalwa) [inline index.html]: ## naming wrong
you are using `card__image` outside its block

File: index.html [hunk 25]
```diff
@@ -0,0 +1,453 @@
...
+                </div>
+                <div class="testimony__user-title">
+                    CEO
+                </div>
+            </div>
+            <div class="testimony__user-card">
+                <div class="testimony__coments">
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+                </div>
+                <img src="./assets/image/marketing.png" class="card__image" alt="user profile">
```
Comment (IbrahimBagalwa) [inline index.html]: `card__image` here as well

File: index.html [hunk 26]
```diff
@@ -0,0 +1,453 @@
...
+                </div>
+                <div class="testimony__user-title">
+                    Marketing Manager
+                </div>
+            </div>
+            <div class="testimony__user-card">
+                <div class="testimony__coments">
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+                </div>
+                <img src="./assets/image/cto.png" class="card__image" alt="user profile">
```
Comment (IbrahimBagalwa) [inline index.html]: `card__image` outside its block

File: page/product.html [hunk 27]
```diff
@@ -0,0 +1,370 @@
...
+            </div>
+        </div>
+    </div>
+
+    <div class="brands">
+        <img class="brand__logo" src="../assets/svgs/brands/brand1.svg" alt="brand icon">
+        <img class="brand__logo" src="../assets/svgs/brands/brand2.svg" alt="brand icon">
+        <img class="brand__logo" src="../assets/svgs/brands/brand3.svg" alt="brand icon">
+        <img class="brand__logo" src="../assets/svgs/brands/brand4.svg" alt="brand icon">
+        <img class="brand__logo" src="../assets/svgs/brands/brand5.svg" alt="brand icon">
```
Comment (IbrahimBagalwa) [inline page/product.html]: ## name wrong
`brand__logo` you are using element outside its block

File: page/product.html [hunk 28]
```diff
@@ -0,0 +1,370 @@
...
+        <div class="scale__card">
+            <div class="scale__header scale__header--posistion-left">
+                <div class="scale__higlight">
+                    Marketing insights
+                </div>
+                <div class="scale__header-text">
+                    Data-driven client feedback
+                </div>
+            </div>
+            <div class="scale-card__text">
```
Comment (IbrahimBagalwa) [inline page/product.html]: `scale-card__text`  you are using an element outside its block

File: page/product.html [hunk 29]
```diff
@@ -0,0 +1,370 @@
...
+    <div class="solution">
+        <div class="solution__header">
+            <div class="solution__title">
+                Features
+            </div>
+        </div>
+        <div class="solution__container solution__container--bg-secondary">
+            <div class="solution__quality">
+                <img src="../assets/svgs/setting.svg" class="solution__quality-icon" alt="check">
+                <div class="solition__quality-title">
```
Comment (IbrahimBagalwa) [inline page/product.html]: ## name wrong
you are using `solition__quality-title` outside its block

File: page/product.html [hunk 30]
```diff
@@ -0,0 +1,370 @@
...
+                    Advanced 256-bit encryption
+                </div>
+                <div class="solition__description">
```
Comment (IbrahimBagalwa) [inline page/product.html]: `solition__description` here as well

File: page/product.html [hunk 31]
```diff
@@ -0,0 +1,370 @@
...
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+                </div>
+            </div>
+            <div class="quality quality--middle">
```
Comment (IbrahimBagalwa) [inline page/product.html]: ## prohibation nested block
`quality` nested block is not a good practice

File: page/product.html [hunk 32]
```diff
@@ -0,0 +1,370 @@
...
+                <img src="../assets/svgs/chatroom.svg" class="solution__quality-icon" alt="check">
+                <div class="solition__quality-title">
+                    Simple collaboration tools
+                </div>
+                <div class="solition__description">
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+                </div>
```
Comment (IbrahimBagalwa) [inline page/product.html:231]: `solition__quality-title` `solition__description` outside its block

File: page/product.html [hunk 33]
```diff
@@ -0,0 +1,370 @@
...
+            </div>
+            <div class="quality">
```
Comment (IbrahimBagalwa) [inline page/product.html]: ## prohibation nested block
`quality` block inside an other block is not a best practice

File: page/product.html [hunk 34]
```diff
@@ -0,0 +1,370 @@
...
+                <img src="../assets/svgs/bot.svg" class="solution__quality-icon" alt="check">
+                <div class="solition__quality-title">
+                    Customizable AI features
+                </div>
+                <div class="solition__description">
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+                </div>
```
Comment (IbrahimBagalwa) [inline page/product.html:240]: `solition__quality-title` `solition__description` you are using an element ouside its block

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
## Request Changes

apply changes, then re-request for the review.
### thanks

File: index.html [hunk 35]
```diff
@@ -0,0 +1,453 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./dist/output.css">
+    <title>Finesweet</title>
+</head>
+<body>
+    <div class="header">
+        <div class="header__links">
```
Comment (edson-pro) [inline index.html]: change header links div may to to header__nav bcs i thought it was the actual links

File: index.html [hunk 36]
```diff
@@ -0,0 +1,453 @@
...
+                    <a class="header__link" href="./page/contact.html">Contact</a>
+                </li>
+                <li class="header__navigation-item" >
+                    <button class="header__button">
+                        View Pricing
+                    </button>
+                </li>
+            </ul>
+        </div>
+        <div class="header__content">
```
Comment (edson-pro) [inline index.html:42]: change header__content to header__hero may be it can be understandable

File: index.html [hunk 37]
```diff
@@ -0,0 +1,453 @@
...
+                High-quality
+            </div>
+            <div class="solution__title">
+                We have the Best Solution <br> for your Business
+            </div>
+        </div>
+        <div class="solution__container solution__container--bg-primary">
+            <div class="solution__quality">
+                <img src="./assets/svgs/check.svg" class="solution__quality-icon" alt="check">
+                <div class="solition__quality-title">
```
Comment (edson-pro) [inline index.html]: solition__quality-title has no block called like this

File: index.html [hunk 38]
```diff
@@ -0,0 +1,453 @@
...
+    <div class="solution">
+        <div class="solution__header">
+            <div class="solution__header-title  solution__header-title--size-small ">
+                Features
+            </div>
+        </div>
+        <div class="solution__container solution__container--bg-secondary">
+            <div class="solution__quality">
+                <img src="./assets/svgs/chev.svg" class="solution__quality-icon" alt="check">
+                <div class="solition__quality-title">
```
Comment (edson-pro) [inline index.html]: & here

File: index.html [hunk 39]
```diff
@@ -0,0 +1,453 @@
...
+                        Subscribe to our Newsletter                       
+                    </li>
+                    <li class="footer__input-group">
+                    </li>
+                </ul>
+            </div>
+            <div class="footer__social-links">
+                <img class="footer__line" src="./assets/svgs/line.svg" alt="line">
+                <div class="social-links">
+                    <div class="footer__copylight">
```
Comment (edson-pro) [inline index.html:440]: footer__copylight is not a social link

File: index.html [hunk 40]
```diff
@@ -0,0 +1,453 @@
...
+                    <li class="footer__links-header">
+                        Subscribe to our Newsletter                       
+                    </li>
+                    <li class="footer__input-group">
+                    </li>
+                </ul>
+            </div>
+            <div class="footer__social-links">
+                <img class="footer__line" src="./assets/svgs/line.svg" alt="line">
+                <div class="social-links">
```
Comment (edson-pro) [inline index.html]: you started social-links un nesessary brock

**Review by edson-pro — CHANGES_REQUESTED** [review]
apply the change

File: index.html (hunk 37 above)
Comment (chrisostome01) [reply index.html]: The block is located  to is called solution

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job @sezeranoJchrisostome 👍 

now it LGTM 👍 
Keep it up :)
### thanks

**Review by edson-pro — APPROVED** [review]
LGTM

File: index.html (hunk 36 above)
Comment (chrisostome01) [reply index.html:42]: header__hero I guess this would would work something like text

File: index.html [hunk 41]
```diff
@@ -0,0 +1,453 @@
...
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.
+            </div>
+        </div>
+        <div class="scale__card">
+            <div class="scale__card-item">
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/charticon.svg" alt="arrow right">
+                    </div>
+                    <div class="scale-card__content">
```
Comment (musayann) [inline index.html]: This class name doesn't match the block class name style

File: index.html [hunk 42]
```diff
@@ -0,0 +1,453 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="scale__card-item">
+                <div class="scale__card-body">
+                    <div class="scale__card-circle">
+                        <img src="./assets/svgs/usericon.svg" alt="arrow right">
+                    </div>
+                    <div class="scale__card-content">
+                        <div class="scale_card-title">
```
Comment (musayann) [inline index.html]: This class name doesn't match the block class name style

**Review by musayann — CHANGES_REQUESTED** [review]
Looks great, just a couple of issues to address

**Review by edson-pro — APPROVED** [review]
GOOD

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job @sezeranoJchrisostome 👍 

LGTM 👍

**Pull request merged**